pkg/prompts/prompts.go

```diff
@@ -141,28 +142,10 @@ func NoBlankStringValidator(s string) error {
 
 // Validator for App name
 func appNameValidator(name string) error {
-	if name == "" {
-		return fmt.Errorf("application name cannot be empty")
+	errors := validation.IsDNS1123Label(name)
```

Comment:
💡 Note only please for archive sake: IsDNS1123Label tests for a string that conforms to the definition of a label in DNS ([RFC 1123](https://rfc-editor.org/rfc/rfc1123.html)).